content/_data/changelogs/weekly.yml

```diff
@@ -8209,6 +8209,276 @@
+  - type: rfe
+    category: rfe
+    pull: 4683
+    issue: 62014
+    authors:
+    - daniel-beck
+    message: |-
+      Add the ability to filter out environment variables for Shell and Windows batch build steps.
+  - type: rfe
+    category: rfe
+    pull: 4835
+    issue: 63002
+    authors:
+    - fqueiruga
+    message: |-
+      Revamp the tabs and tables with better spacing and a consistent color palette.
+  - type: rfe
+    category: rfe
+    pull: 4748
+    issue: 61806
+    authors:
+    - 95jonpet
+    - jglick
+    message: |-
+      Remove page generation timestamp from the footer.
+  - type: rfe
+    category: rfe
+    pull: 4840
+    issue: 25259
+    authors:
+    - tizki
+    message: |-
+      Add sorting to Old Data management tables.
+  - type: rfe
+    category: rfe
+    pull: 4834
+    authors:
+    - stellargo
+    message: |-
+      Allow fingerprint storage engine to be selected from the configuration page.
+    references:
+      - issue: 63022
+      - url: https://github.com/jenkinsci/jep/tree/master/jep/226
+        title: JEP-226
+  - type: rfe
+    category: rfe
+    pull: 4073
+    issue: 59136
+    authors:
+    - jetersen
+    - daniel-beck
+    message: |-
+      Show warnings for <b>deprecated</b> plugins in the update manager and administrative monitors.
+  - type: rfe
+    category: rfe
+    pull: 4820
+    authors:
+    - timja
+    message: |-
+      Update Jenkins form styling in preparation for changing form layout.
+    references:
+      - issue: 56109
+      - url: https://issues.jenkins-ci.org/browse/JENKINS-62437
+        title: "Configuration UI Accessibility: Tables to Divs migration"
+  - type: rfe
+    category: rfe
+    pull: 4841
+    issue: 61806
+    authors:
+    - jglick
+    message: |-
+      Show hints about the current time on the server’s zone when configuring your <b>User Defined Time Zone</b>.
+  - type: bug
+    category: bug
+    pull: 4833
+    issue: 62723
+    authors:
+    - jglick
+    - Zastai
+    message: |-
+      Fix <code>IllegalArgumentException: Method not found</code> error caused by misbehaviour in <code>Util.isOverridden()</code> (regression in 2.241).
+  - type: bug
+    category: bug
+    pull: 4830
+    authors:
+    - jglick
+    message: |-
+      Avoid a stack trace when an older version of glibc was used by an agent.
+  - type: bug
+    category: bug
+    pull: 4839
+    issue: 63014
+    authors:
+    - jglick
+    message: |-
+      Remove the fallback Jenkins URL from the JNLP launch file so that WebSocket agents can be connected over Java Web Start.
+  - type: rfe
+    category: developer
+    pull: 4683
+    issue: 62014
+    authors:
+    - daniel-beck
+    message: |-
+      Developer: Add new extension points to define build step environment filters (currently in beta).
+  - type: rfe
+    category: developer
+    pull: 4842
+    references:
+      - url: https://github.com/jenkinsci/dark-theme
```

Comment:
Why link to GitHub instead of the plugin site?

Reply:
Historical, it didn’t use to be a plugin so all the change logs have used the GitHub repo but yeah plugin site would be good